### ISSUE TITLE
Cannot create an environment with a Classic Load Balancer

### ISSUE THREAD

**Issue by AndyRN** [aws/aws-elastic-beanstalk-cli#41]
### Description

Using a configuration template, such as this `example.cfg.yml`, to create environments within Elastic Beanstalk by running `eb create example --cfg example --sample`, they are always created with an application load balancer.
```
OptionSettings:
  aws:autoscaling:launchconfiguration:
    IamInstanceProfile: aws-elasticbeanstalk-ec2-role
  aws:elasticbeanstalk:environment:
    LoadBalancerType: classic
  aws:ec2:instances:
    InstanceTypes: t2.micro
EnvironmentTier:
  Type: Standard
  Name: WebServer
AWSConfigurationTemplateVersion: 1.1.0.0
```
*In order to use this template, you'll need to specify the `aws:ec2:vpc` namespace if you do not have a default VPC within your AWS account*

The `LoadBalancerType` has been set for transparency, but the documentation states that it should default to `classic` anyway: https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-elasticbeanstalkenvironment.

With this in mind, it appears that the above template successfully creates an environment with a classic load balancer when running an earlier version of the `ebcli`.

### Steps to reproduce

*Assuming a folder structure as defined here: https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environment-configuration-methods-during.html#configuration-options-during-ebcli-savedconfig*

**With EB CLI 3.16.0 installed**
* [ ] Run `eb create example-v-3-16-0 --cfg example --sample`.
* [ ] On the AWS console, check the configuration of the created environment. It will have a classic load balancer.

**With EB CLI 3.18.2 installed**
* [ ] Run `eb create example-v-3-18-2 --cfg example --sample`.
* [ ] On the AWS console, check the configuration of the created environment. It will have an application load balancer.

### Observed result

![image](https://user-images.githubusercontent.com/1238199/89527389-2a914b80-d7e1-11ea-97e0-178b144002a5.png)

### Expected result

![image](https://user-images.githubusercontent.com/1238199/89524834-26632f00-d7dd-11ea-8eb0-1c44e75b5b2d.png)

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Microsoft Windows 10 Pro - Version 10.0.18363 Build 18363
2. EBCLI version: EB CLI 3.18.2 (Python 3.8.0)

**Comment by Palakpatel67** [comment]
Hi @AndyRN, thanks for reporting.

This bug was introduced in the latest release, so if you would like to continue to use this functionality while waiting for a fix to be released you can downgrade awsebcli to version 3.18.1. 

Leaving this issue open while the fix is pending release.

**Comment by Palakpatel67** [comment]
this should be fixed in [V3.19.0](https://pypi.org/project/awsebcli/)

**Comment by benjohnstonavco** [comment]
Looks as though this has been re-introduced in v3.20.5 in comparison to v3.19.2. Can this be re-opened?

**Comment by yohanthshetty** [comment]
Thanks @benjohnstonavco for bringing this to our notice. Tea will verify and reopen if needed.

**Comment by benmj87** [comment]
This is still happening, did it get reviewed?

**Comment by Diazole** [comment]
This issue was reintroduced by [7bf2f7f](https://github.com/aws/aws-elastic-beanstalk-cli/commit/7bf2f7fc67c3e295714b089e8ab0615d825f5a71#diff-0e5fd502090ae4c475ad3783792164f69bb403bc3db910e233c4d27f044e135aR548).

It was then fixed again by #501, however @rahulrajaram reintroduced it again with #522.

It's currently not possible to create a classic load balancer due to this issue.
